NExtends/Expressions/KeepTypeInPlaceVisitor.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using static NExtends.Expressions.ExpressionHelper;
+
+namespace NExtends.Expressions
+{
+	public class KeepTypeInPlaceVisitor<TSource, TKeep> : ExpressionVisitor
+	{
+		private readonly Type _lambdaToKeep;
+		private readonly ParameterExpression _newParameter;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the '_newParameter' field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1450)